custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt

```diff
@@ -49,6 +49,9 @@ class WorkAnIssue(
     }
 
     private fun edit(issuePage: AbstractIssuePage) {
+        meter.measure(ActionType("Edit Issue Description") {Unit}) {
+            issuePage.editDescription("updated")
+        }
```

Comment:
You can validate how it looks like thanks to `TakeScreenshotHook`. It should generate a sample of screenshots per every `ActionType`. AFAIK it's only used in `JiraCloudScenario` right now, so you can include it in `JiraDcScenario` too.